app/build.gradle.kts

```diff
@@ -92,6 +92,15 @@ android {
 
 play {
   enabled.set(true)
+  val jsonData = System.getenv("google_json")
+  val jsonFile: File = file("../google.json")
```

Comment:
Why `../`?

Reply:
We have already ```google.json``` file outside from ```app``` directory and ```app/build.gradle.kts``` file inside the ```app directory``` so that's why we are using ```../``` for writing on the same file instead of creating new file inside the ```app``` directory.

Reply:
What is this google.json in our app directory? It's not in the git!

Reply:
Why it is still called `google.json` and not `playstore.json`?

Reply:
Previously we are using the ```google.json``` file it looks like Firebase json file (i think previously we are using publication with firebase) and we can not change the firebase json file name, that's why i have not renamed this yet. https://github.com/kiwix/kiwix-android/blob/838b2175961155637dc84ccb525301d7e0202641/google.json#L4

But now we are using GPP so i have renamed this file as ```playstore.json```.